crypto/asn1/bio_asn1.c

```diff
@@ -16,6 +16,7 @@
 #include <string.h>
 #include <internal/bio.h>
 #include <openssl/asn1.h>
+#include "e_os.h"
```

Comment:
Ok, but ossl_assert is already in e_os.h, i.e. this PR is just using it, so if you want it to be moved then that should probably be a separate PR. Are you objecting to *this* PR in any way?